raiden_contracts/contracts/raiden/TokenNetwork.sol

```diff
@@ -31,23 +31,27 @@ contract TokenNetwork is Utils {
         115792089237316195423570985008687907853269984665640564039457584007913129639935
     );
 
+    /*LIMITED-VERSION-START*/
```

Comment:
Why not change this to something like `TokenNetwork_template.sol` and then use the tool to generate two different files? This could be a bit cleaner and we could have the code without those ugly "preprocessor" markers.

Reply:
That's the way to go!

Reply:
* http://mustache.github.io/
* http://jinja.pocoo.org/ 
* https://docs.python.org/3.4/library/string.html#template-strings doesn't seem to work for this usage.